package.json

```diff
@@ -6,39 +6,40 @@
   "dependencies": {
     "@chakra-ui/next-js": "^2.1.5",
     "@chakra-ui/react": "^2.8.1",
-    "@cosmjs/cosmwasm-stargate": "^0.31.3",
-    "@cosmjs/stargate": "^0.31.3",
-    "@cosmos-kit/core": "^2.7.2",
-    "@cosmos-kit/cosmostation": "^2.4.4",
-    "@cosmos-kit/keplr": "^2.4.4",
-    "@cosmos-kit/leap": "^2.4.3",
-    "@cosmos-kit/react": "^2.5.3",
+    "@cosmjs/cosmwasm-stargate": "^0.32.3",
+    "@cosmjs/stargate": "^0.32.3",
+    "@cosmos-kit/core": "^2.12.0",
+    "@cosmos-kit/cosmostation": "^2.10.0",
+    "@cosmos-kit/keplr": "^2.11.0",
+    "@cosmos-kit/leap": "^2.11.0",
+    "@cosmos-kit/react": "^2.15.0",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@headlessui/react": "^1.7.14",
     "@hyperlane-xyz/registry": "2.1.0",
     "@hyperlane-xyz/sdk": "3.13.0",
     "@hyperlane-xyz/utils": "3.13.0",
     "@hyperlane-xyz/widgets": "3.13.1",
+    "@interchain-ui/react": "^1.23.16",
```

Comment:
I can't quite remember but I _think_ there may have been a reason I didn't include this, like to avoid it's inclusion in the bundle when it's not really necessary. Can you confirm it's a necessary addition?